### ISSUE TITLE
Nimbus Consensus Migration Stage 0

### ISSUE THREAD

**Issue by Garandor** [Manta-Network/Manta#668]
This is phase 0 of the phased Nimbus consensus rollout described in #486 

Nimbus Consensus uses different validator keys than our currently used Aura

To allow collator runners to upgrade their nodes in preparation for the switch to Nimbus, we add the required key to the KMA and DOL runtimes.
For futureproofing, we also enable a VRF key type in addition to the Nimbus key
( see https://github.com/PureStake/moonbeam/pull/1480 )
This will allow us to change to a VRF lottery style block producer election without asking collators to recreate their keychain again.

In summary:
- [ ] Merge code with wrapped nimbus opaque key with OneSessionHandler and noop on_new_session #646 
- [ ] Issue runtime upgrade to v3.2.1
- [ ] publish community post to call set_keys in advance of nimbus enabling
- [ ] update keys of manta-run collators
  - [ ] dolphin
  - [ ] calamari
